lib/common/output_log.c

```diff
@@ -82,12 +89,16 @@ log_reset(pcmk__output_t *out) {
 
 static void
 log_version(pcmk__output_t *out, bool extended) {
+    private_data_t *priv = out->priv;
+
+    CRM_ASSERT(priv != NULL);
```

Comment:
If we're going to assert this we may as well assert out != NULL && out->priv != NULL before grabbing priv

Reply:
Fixed, and I'll also go through the rest of the formatted output code to make sure this is being done consistently.  I'll add another patch to #2326 if necessary.